### ISSUE TITLE
Delta Logger blocks AE2 autocrafting

### ISSUE THREAD

**Issue by Technici4n** [fabricservertools/DeltaLogger#37]
**Describe the bug**
Delta Logger blocks AE2 crafting through the terminal.

**To Reproduce**
Instal Delta Logger along with AE2, setup a basic system and an encoded pattern, and attempt to autocraft something. The autocrafting GUI will say `Crafting Plan - Calculating Please Wait`.

**Expected behavior**
Show the crafting plan and allow it to be started.

**Screenshots**
![image](https://user-images.githubusercontent.com/13494793/110173830-810a4b00-7dff-11eb-88bd-8532c5016b6e.png)

**Logs**
https://gist.github.com/Technici4n/04edf8b18a324fd8f77ff64a26f79229

Delta Logger NPE: https://github.com/fabricservertools/DeltaLogger/blob/0409f3fe7701697f43070e2b6da8205e214f6b0c/src/main/java/com/github/fabricservertools/deltalogger/listeners/PlayerEventListener.java#L60.

Basically Delta Logger is trying to `getSlot(0)`, but the crafting plan screen handler has no slots, thus leading to an exception which prevents the normal behavior.

**Comment by yitzy299** [comment]
What's the AE2 modid, for checking whether it's loaded against the loader API?

**Comment by Technici4n** [comment]
pretty sure it's `appliedenergistics2`, but imo you should just check if the screen handler has at least one slot - probably needs an accessor mixin.
